core/src/test/scala/fs2/TestUtil.scala

```diff
@@ -32,7 +34,7 @@ trait TestUtil {
     catch {
       case e: InterruptedException => throw e
       case e: TimeoutException => throw e
-      case e: Throwable => ()
+      case NonFatal(e) => ()
```

Comment:
Perhaps. Akka has such behavior but lets you disable it via configuration. We might want to do the same if we decide to shutdown JVM.